### ISSUE TITLE
Server returns 500 error after revisiting days later

### ISSUE THREAD

**Issue by manicmachine** [Rvndyr/MultiMon-frontend#8]
Upon revisiting the site several days later, I found that I wasn't able to see my profile pic, name, or channels listed. Looking at the console it looks like the server was returning a 500 error

![Screen Shot 2021-07-22 at 9 36 47 AM](https://user-images.githubusercontent.com/11064500/126658744-34b17751-2925-4816-a64c-472cd8339f5b.png)

It appears to be cookie related as the issue was resolved by logging out and back in. Also, it didn't occur within a Private/Incognito window.

**Comment by Rvndyr** [comment]
Yeah, I need to figure out what are the best practices for cookies expiration and when to actually delete the local storage token. More research needs to be done on my end for this.
